pptx_parser.py

```diff
@@ -0,0 +1,29 @@
+from pptx import Presentation
+
+
+class PptxParser:
+    """
+    Parsing a pptx file and yield the slide's text and index number.
+    """
+    @classmethod
+    def parse(cls, filepath):
+        """
+        Receives a path to a pptx file and parse the presentation.
+        :param filepath: str path to a pptx file
+        :yield: slide's text and index number
+        """
+        prs = Presentation(filepath)
+        for index, slide in enumerate(prs.slides):
```

Comment:
You made 4 for canonical loops, 
That's too much - think of a better way to implement your logic.